lab_5_scrapper/scrapper.py

```diff
@@ -213,4 +351,9 @@ def main() -> None:
 
 
 if __name__ == "__main__":
-    main()
+    #main()
```

Comment:
all script logic should be placed in main, even some debug code

Reply:
Done